proposals/0016-simd-application-fees.md

```diff
@@ -0,0 +1,438 @@
+---
+simd: '0016'
+title: Application Fees
+authors:
+  - Godmode Galactus (Mango Markets)
+  - Maximilian Schneider (Mango Markets)
+category: Standard
+type: Core, Fees
+status: Draft
+created: 2022-12-23
+feature:
+---
+
+
+## Summary
+
+This SIMD will discuss additional fees called Application Fees. These fees are
+decided and set by the dapp developer to interact with the dapp. Dapp developers
+then can decide to rebate these fees if a user interacts with the dapp as
+intended and disincentivize the users which do not. These fees will be applied
+even if the transaction eventually fails and are collected on the writable
+account. The owner of the account can do lamport transfer to recover these
+fees. So instead of fees going to the validator, these fees go to the **dapp
+developers**. It will be dapp developer's responsibility to advertise the
+required application fees to its users.
+
+Discussion for the issue: <https://github.com/solana-labs/solana/issues/21883>
```

Comment:
SIMDs should stand alone. Please include any salient outcomes from that discussion inline